todo_app/app.py

```diff
@@ -8,4 +9,13 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template("index.html", items=items)
+
+
+@app.route('/add_items', methods=["POST"])
+def create_items():
```

Comment:
It is okay for the function name to be different from the URL route - sometimes the function name needs to be more descriptive than the route should/can be.

That said, I think the route and function names should both either start with `create` or `add`, there is no benefit to the difference.

Reply:
Noted, thank you!

I think I got a bit confused wrt the `add_item` function that was a part of the `session_items` script. Will resolve to a single name for the route and the function.